pkg/kn/flags/channel_types.go

```diff
@@ -93,21 +93,24 @@ func (i *ChannelRef) Add(f *pflag.FlagSet) {
 		"channel",
 		"",
 		"Specify the channel to subscribe to, in the format '--channel Group:Version:Kind:Name'. "+
-			"You can use channel type aliases from kn config with this flag. "+
-			"You can also refer inbuilt channel type InMemoryChannel using alias 'imc' like '--type imc:CHANNEL_NAME'. "+
-			"Examples: '--channel messaging.knative.dev:v1alpha1:KafkaChannel:k1' for specifying explicit Group:Version:Kind:Name.")
+			"You can refer channel type aliases from kn config with this flag. "+
+			"You can also refer inbuilt channel type aliases 'imcv1beta1' or 'imc'. "+
+			"Examples: '--channel messaging.knative.dev:v1alpha1:KafkaChannel:k1' for specifying explicit Group:Version:Kind:Name. "+
+			"If a prefix is not provided, it is considered as Channel(messaging.knative.dev/v1beta1).")
```

Comment:
```suggestion
			"If no prefix is provided, the default channel is used.")
```

Reply:
Or maybe even better rephrase to mention the default channel first, like in:

```
Specify the channel to subscribe to. For the default channel, just use the name (e.g. "mychannel"). A mapped channel type like "imc" can be used as a prefix (e.g. "imc:mychannel"). Finally you can specify the full coordinates to the referenced channel with Group:Version:Kind:Name (e.g. "messaging.knative.dev:v1alpha1:KafkaChannel:mychannel")
```

Reply:
thanks! I've used the second suggestion.